docs/3.0.1.md

```diff
@@ -0,0 +1,32 @@
+# Percona build of ProxySQL 3.0.1 and ProxySQL admin tools (2025-06-12)
+
+--8<--- "rn-block-top.md"
```

Comment:
This snippet contains a mention of ProxySQL Admin 2

[ProxySQL Admin 2](https://proxysql-3-0-1-release-notes-566.onrender.com/proxysql-admin-tool-functions.html) supports Percona XtraDB Cluster without custom scripts.